apps/backend/src/modules/activity-logs/activity-log.service.ts

```diff
@@ -148,7 +148,10 @@ class ActivityLogService implements Service {
 
 		const allContributors = await (projectId
 			? this.contributorService.findAllByProjectId(Number(projectId))
-			: this.contributorService.findAll());
+			: this.contributorService.findAll({
+					page: 1,
+					pageSize: Number.MAX_SAFE_INTEGER,
+				}));
```

Comment:
Here we either need to separate these two methods in the contributors controller and repository, to have one with pagination and one without or add pagination to the analytics table too. let's move on with the first option in this PR.